src/stateMachines/ieee8021xConfiguration.ts

```diff
@@ -0,0 +1,465 @@
+/*********************************************************************
+ * Copyright (c) Intel Corporation 2022
+ * SPDX-License-Identifier: Apache-2.0
+ **********************************************************************/
+
+import { type IPS, type AMT } from '@open-amt-cloud-toolkit/wsman-messages'
+import { assign, createMachine, send } from 'xstate'
+import { type HttpHandler } from '../HttpHandler'
+import Logger from '../Logger'
+import { type AMTConfiguration } from '../models'
+import { devices } from '../WebSocketListener'
+import { Error } from './error'
+import { Configurator } from '../Configurator'
+import { DbCreatorFactory } from '../factories/DbCreatorFactory'
+import { invokeEnterpriseAssistantCall, invokeWsmanCall } from './common'
+import { UNEXPECTED_PARSE_ERROR } from '../utils/constants'
+import { type EnterpriseAssistantMessage } from '../WSEnterpriseAssistantListener'
+
+export interface IEEE8021xConfigContext {
+  amtProfile: AMTConfiguration
+  ieee8021xProfile: any
+  message: any
+  clientId: string
+  xmlMessage: any
+  statusMessage: string
+  httpHandler: HttpHandler
+  targetAfterError: string
+  errorMessage: string
+  retryCount?: number
+  ips?: IPS.Messages
+  amt?: AMT.Messages
+  eaResponse: any
+  addTrustedRootCertResponse: any
+  addCertResponse: any
+  keyPairHandle?: string
```

Comment:
in the class reference InstanceID is not the same thing as a key handle. is there some reason keyHandle is used in the context? it is always referenced/used as the key pair instance id so it is confusing to me.